src/custom/Markdown/style.tsx

```diff
@@ -9,12 +10,12 @@ export const StyledMarkdown = styled('a')(({ theme }) => ({
 }));
 
 export const StyledMarkdownP = styled('p')(({ theme }) => ({
-  color: theme.palette.text.default,
+  color: theme.palette.text.inverse || text.inverse,
```

Comment:
also added a conditional token like this so that in anycase theme doesn't effect there will not be any bug with the color

Reply:
No use of text constant white?

Reply:
fixed